linux/deCONZ/usr/bin/deCONZ-ikea-ota-download.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+curl 'http://fw.ota.homesmart.ikea.net/feed/version_info.json' | jq -r '[.[] | .fw_binary_url] | .[]'  > $HOME/files.txt
```

Comment:
Nice addition. It seems this URL works with HTTPS as well, which would be preferable if it works, right?

Reply:
I see now that the firmware filenames include `.signed` and figured there might be other mechanisms to prevent mitm attacks etc., so perhaps I was a bit overzealous in my previous comment.

Reply:
> Nice addition. It seems this URL works with HTTPS as well, which would be preferable if it works, right?

You're right. https works as well. I just did a copy and paste without thinking about it :-).

Reply:
What I discovered is that the response JSON refers to additional resources served over (unsecured) HTTP, so it might actually make the most sense to keep the same protocol for the first request as well to avoid a false sense of security? (This would be inline with how Debian and others go about this, trusting the signatures of packages rather than the transfer mechanism to ensure integrity...)

Reply:
No problem. I reseted the changes.